draft-ietf-jsonpath-base.md

```diff
@@ -1535,36 +1523,80 @@ A function expression is well-typed if all of the following are true:
    * The argument is a literal primitive value and the defined type of the parameter is `ValueType`.
    * The argument is a Singular Path or `filter-path` (which includes
      Singular Paths), or a function expression with declared result
-     type `NodesType`.  Where the declared type of the parameter is
-     not `NodesType`, a conversion applies.
+     type `NodesType`.
```

Comment:
This bullet is confusing because it doesn't way what parameter type(s) accept paths and `NodesType` functions. Should it say that the parameter type is `NodesType`?

Reply:
Right.  Cleaned this up some (the third bullet also was half-redundant with the first).